### ISSUE TITLE
"Travel Review" text color in Reviews by Year chart

### ISSUE THREAD

**Issue by eugenern** [andyklimczak/TheReportOfTheWeek-client#1]
Very minor, but for some reason in the Reviews by Year chart, if you hover over the 2018 bar first and then next hover over the 2017 bar, the "Travel Review" text is yellow instead of black. The text is its intended black under any other circumstances.

**Comment by andyklimczak** [comment]
good catch, will investigate if it's how I'm setting the keys in the `categories.map` loop in `CategoryYearlyStacked` component or a bug with the charting library!
